doc/run_experiment.rst

```diff
@@ -406,6 +406,26 @@ example, if you wanted to collapse the labels ``beagle`` and ``dachsund`` into a
 
 Any labels not included in the dictionary will be left untouched.
 
+One other use case for ``class_map`` is to deal with classification labels that
+would be converted to ``float`` improperly. All ``Reader`` sub-classes use the
+``safe_float`` function internally to read labels. This function tries to
+convert a single label first to ``int``, then to ``float``. If neither
+conversion is possible, the label remains a ``str``. Thus, care must be taken
+to ensure that labels do not get converted in unexpected ways. For example,
+consider the situation where there are classification labels that are a mixture
+of ``int``-converting and ``float``-converting labels:
+
+.. code-block:: python
+    import numpy as np
```

Comment:
You need an empty line between the `code-block` directive and the actual code - it causes an error on my machine otherwise.